aeon/schema/social_03.py

```diff
@@ -6,3 +8,19 @@ class Pose(Stream):
 
     def __init__(self, path):
         super().__init__(_reader.Pose(f"{path}_202_*"))
+
+
+class EnvActiveConfigReader(_reader.JsonList):
```

Comment:
Pulling out specific attributes from the JSON into columns might be a common enough operation that we could provide it at the level of the general reader, i.e. provide a `columns` property to specify which values to pull out from each record.

Reply:
For example, we can do something like this:

```
class JsonList(Reader):
    """Extracts data from json list (.jsonl) files, where the key "seconds"
    stores the Aeon timestamp, in seconds.
    """

    def __init__(self, pattern, columns=None, col_tuples=[], extension="jsonl"):
        super().__init__(pattern, columns, extension)
        self.col_tuples = col_tuples

    def read(self, file):
        """Reads data from the specified jsonl file."""
        with open(file, "r") as f:
            df = pd.read_json(f, lines=True)
        df.set_index("seconds", inplace=True)
        for new_col, from_attr in self.col_tuples:
            df[new_col] = df.apply(lambda x: x[from_attr])
        return df
```

Reply:
(with better naming... `col_tuples` is a horrible name)

Reply:
@glopesdev I don't have a strong preference on whether the pulling out of the column names occurs (whether in `JsonList` or in the lower-level Reader), so I guess you can make a final call here and then we can just go with that?